week2/exercises/src/components/friends/FriendProfile.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+const FriendProfile = props => {
+  return (
+    <div>
+      <ul key={Math.random()} className="list">
```

Comment:
Key, do we have an array surround this `ul` element? 🤔
```suggestion
      <ul className="list">
```

Reply:
Off course not :)